pkg/operator/starter.go

```diff
@@ -0,0 +1,95 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cert-manager-operator/pkg/controller/deployment"
```

Comment:
I like to take the local imports into their own separate block, it sometimes makes rebasing easier, especially on someone else's changes

Reply:
sure thing :+1: